crates/credentials/src/verifiable_credential.rs

```diff
@@ -130,6 +130,52 @@ pub struct VerifiableCredential {
     pub credential_subject: CredentialSubject,
 }
 
+impl VerifiableCredential {
+    fn into_jwt_payload_verifiable_credential(self) -> JwtPayloadVerifiableCredential {
+        JwtPayloadVerifiableCredential {
+            context: self.context,
+            id: Some(self.id),
+            r#type: self.r#type,
+            issuer: Some(self.issuer),
+            issuance_date: Some(self.issuance_date),
+            expiration_date: self.expiration_date,
+            credential_subject: Some(self.credential_subject),
+        }
+    }
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct JwtPayloadVerifiableCredential {
```

Comment:
Should this be a `pub struct`? It's used as a field of `pub struct VcJwtClaims`. It would be weird for a consumer to work with a `VcJwtClaims` but not be able to work easily with one of its fields. If it should be `pub`, can you add a doc comment explaining why `JwtPayloadVerifiableCredential` is a distinct type from `VerifiableCredential` and how the two should be used distinctly.

Reply:
yup probably so! I'll make it pub